tests/testthat/tests_03_predict_sex.R

```diff
@@ -1,6 +1,6 @@
 library(testthat)
 
-setwd("/workdir/")
+setwd("/workspaces/dimorfismo/")
```

Comment:
El directorio `/workspaces/dimorfismo/` es particular a Visual Studio Code. Los miércoles yo uso Atom en lugar de Visual Studio Code. Creo que el cambio que pretendes hacer va en `.devcontainer/devcontainer.json` no en `tests/testthat/tests_03_predict_sex.R`

```suggestion
setwd("/workdir/")
```